ContactList/ContactList/UpdateViewController.swift

```diff
@@ -10,42 +10,30 @@ import UIKit
 
 class UpdateViewController: UIViewController {
     
-    
-    @IBOutlet weak var firstNameTF: UITextField!
-    
-    @IBOutlet weak var lastNameTF: UITextField!
-    
-    @IBOutlet weak var phoneTF: UITextField!
-    
-    @IBOutlet weak var emailTF: UITextField!
-    
-    @IBOutlet weak var imageArea: UIImageView!
-    
-    @IBOutlet weak var saveButton: UIBarButtonItem!
-    
-    @IBOutlet weak var removeButton: UIBarButtonItem!
-    
-    
-    var imageState = ImageEditState.noChanges
+    @IBOutlet private weak var firstNameTF: UITextField!
```

Comment:
Textfields are covered by keyboard even on large screen (see attach).
![Simulator Screen Shot - iPhone 8 - 2019-05-20 at 10 27 15](https://user-images.githubusercontent.com/28065499/58003705-fea97d80-7ae9-11e9-9670-e575ca0aa9c9.png)
I think the situation is even worse on smaller screens.
Expected: User should be able to see his input